### ISSUE TITLE
[Feature Request] Add an official template for H2O LLM Studio

### ISSUE THREAD

**Issue by josephrocca** [runpod/containers#20]
This allows people to fine-tune LLMs and test them without any coding experience. It has become fairly popular and receives regular updates:

https://github.com/h2oai/h2o-llmstudio

**Comment by justinmerrell** [comment]
Agreed, that does look like an excellent container to have. I have added it to our to-do list but PRs are also welcome.
